sql/moz-fx-data-shared-prod/telemetry/active_users_aggregates/view.sql

```diff
@@ -3,6 +3,42 @@ CREATE OR REPLACE VIEW
 AS
 SELECT
   *,
-  `mozfun.norm.os`(os) AS os_grouped
+  dau AS qdau
+FROM `telemetry.active_users_aggregates_mobile`
+UNION ALL
+SELECT
+  segment,
+  attribution_medium,
+  attribution_source,
+  attributed,
+  city,
+  country,
+  distribution_id,
+  first_seen_year,
+  is_default_browser,
+  app_name,
+  channel,
+  os,
+  os_version,
+  os_version_major,
+  os_version_minor,
+  submission_date,
+  language_name,
+  dau,
+  wau,
+  mau,
+  new_profiles,
+  ad_clicks,
+  organic_search_count,
+  search_count,
+  search_with_ads,
+  uri_count,
+  active_hours,
+  app_version,
+  app_version_major,
+  app_version_minor,
+  app_version_patch_revision,
+  app_version_is_major_release,
+  qdau
 FROM
-  `moz-fx-data-shared-prod.telemetry_derived.active_users_aggregates_v1`
+  `moz-fx-data-shared-prod.firefox_desktop.active_users_aggregates`
```

Comment:
It looks like you are replacing the old aua table reference with the new views.  I assume all QA checks and validations are completed, including from the DS side?

Reply:
Hi @winnie,
This PR only changes the view `telemetry.active_users_aggregates` (not the table). Currently we have the validations for [desktop](https://docs.google.com/document/d/1J6VPYXin7Cf9aEItMadFLGooTmCaHhhTUSCaGTNcBlY/edit) and [mobile](https://docs.google.com/document/d/1NY4xFrWr6wf3CYucl-oxFl32-qJmAOL60OrfZpkGsSI/edit#) and [PR3715](https://github.com/mozilla/bigquery-etl/pull/3715) to be merged before this one.

Reply:
Referencing also the [validation story](https://mozilla-hub.atlassian.net/browse/DENG-785) that requires approval from DS. cc/ @bochocki

Reply:
The mobile data looks good! I will update the ticket.